src/screens/AboutUs/index.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import Footer from '../../components/Footer';
+import Navbar from '../../components/Navbar';
+import Sidebar from '../../components/Sidebar';
+import './styles.scss';
+
+export default function AboutUs() {
+  return (
+    <>
```

Comment:
The **AboutUs** should have its own layout. Please check either **Sign up** or **Home** about the implementation. It should be something like `className = layout-about-us`